.prettierrc.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
```

Comment:
I have just noticed that this file was added. I don't think it is needed
@CarolineDenis does prettier work for you even if this file is removed?

I don't think it is looking at this file as long as we have a "prettier" key in package.json. You need to remove either one of them